pos_product_cost_security/tests/test_pos_product_cost.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.tests import HttpCase, tagged
+
+
+@tagged("post_install", "-at_install")
```

Comment:
Can this be avoided? It usually produces random failures on integration environments.

```suggestion
```

Reply:
Well, precisely the PoS tests are a special kind: https://github.com/odoo/odoo/blob/14.0/addons/point_of_sale/tests/test_frontend.py#L474